tests/acceptance/features/apiSharingNg/sharedWithMe.feature

```diff
@@ -2974,4 +2974,444 @@ Feature: an user gets the resources shared to them
         }
       }
     }
-    """
\ No newline at end of file
+    """
+
```

Comment:
tests with `auto-sync` can be moved it it's on feature file. the file has already become too long